classes/PodsInit.php

```diff
@@ -507,7 +507,7 @@ public function register_assets() {
 			true
 		);
 
-		wp_register_script( 'pods-cleditor', PODS_URL . 'ui/js/jquery.cleditor.min.js', array( 'jquery' ), '1.3.0' );
+		wp_register_script( 'pods-cleditor', PODS_URL . "ui/js/cleditor/jquery.cleditor{$suffix_min}.js", array( 'jquery' ), '1.4.5', true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 7bb5203408358d75029ad6dd8c47fd8f</sub>